epochStart/mock/nodesCoordinatorMock.go

```diff
@@ -0,0 +1,155 @@
+package mock
+
+import (
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+// NodesCoordinatorMock -
+type NodesCoordinatorMock struct {
```

Comment:
this is a stub. may implement only the relevant functions

Reply:
done